### DISCUSSION TITLE
Documentation website feedback

### DISCUSSION THREAD

**Discussion by timocov** [tradingview/lightweight-charts#921] [Ideas]
Recently we released [a new documentation website](https://tradingview.github.io/lightweight-charts/). Let's use this discussion to share your ideas/thoughts what you'd like to add/remove/improve/etc to this website.

**Comment by timocov** [comment]
Auto-generated default values in tsdoc (right now we have to write and sync them manually)

**Comment by timocov** [comment]
Allow to use the code sample as a code to execute on the page so our pages could be interactive (instead of using static images)

**Comment by amirsafaricg** [comment]
how about code examples ?

**Reply by timocov** [reply]
Agree. Right now we have some examples in API reference, but nothing interesting there IMO.

**Comment by Skiv1989** [comment]
For this page: https://tradingview.github.io/lightweight-charts/docs/time-scale 
For logical range it says about indexes before first data item, and nothing about indexes after last data item.

**Reply by edew** [reply]
> The starting point of the time scale's logical range is the first data item among all series. Before that point all indexes are negative, starting from that point - positive.

Are you referring to that part @Skiv1989 ?

Logical indexes after the last data point continue increasing.

![image](https://github.com/user-attachments/assets/d5cb1d5c-4ff6-4afd-b283-56c971cb9c2d)

**Reply by Skiv1989** [reply]
Yes, this image is much better
